app/src/hid.c

```diff
@@ -49,6 +49,32 @@ int zmk_hid_unregister_mod(zmk_mod modifier) {
     return 0;
 }
 
+int zmk_hid_register_mods(zmk_mod_flags modifiers) {
+    int err;
+    for (zmk_mod i = 0; i < 8; i++) {
+        if (modifiers & (1 << i)) {
+            err = zmk_hid_register_mod(i);
+            if (err) {
+                return err;
+            }
+        }
+    }
+    return 0;
+}
+
+int zmk_hid_unregister_mods(zmk_mod_flags modifiers) {
+    int err;
+    for (zmk_mod i = 0; i < 8; i++) {
+        if (modifiers & (1 << i)) {
+            err = zmk_hid_unregister_mod(i);
+            if (err) {
```

Comment:
Do we really want to bail on the loop if some entry errors?

Reply:
Nah. There's no way to recover from an error either, so I'll remove the error handling.